administrate-field-lat_lng.gemspec

```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency 'administrate', '~> 0.2.1'
+  spec.add_dependency 'leaflet-rails', '0.7.7'
```

Comment:
Why is this a hard dependency on 0.7.7 and not the 1.x series? Can we please say `'~> 1.0'` instead?